### PR TITLE
fix(button chooser): resolve typing issues

### PR DESCRIPTION
**Pull request by mike-schmid**

### CODE REVIEW

File: src/components/RichTextEditor/serializer/utils/reactCssPropsToCss.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { AnyObject } from '@udecode/plate';
 import { convertCamelCaseToKebabCase } from './convertCamelCaseToKebabCase';
 
-export const reactCssPropsToCss = (props?: any): string => {
+export const reactCssPropsToCss = (props?: AnyObject): string => {
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/serializer/utils/reactCssPropsToCss.ts:6]: 🙈 

Isn't it `DesignTokens`?

File: src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx [hunk 2]
```diff
@@ -11,8 +11,9 @@ import { TextStyleDropdownProps } from './types';
 
 export const TextStyleDropdown = ({ editorId, textStyles = defaultTextStyles }: TextStyleDropdownProps) => {
     const [open, setOpen] = useState(false);
-    const editor = usePlateEditorState(editorId)!;
-    const { designTokens } = useRichTextEditorContext();
+    const editor = usePlateEditorState(editorId);
+    const context = useRichTextEditorContext();
+    const designTokens = context.designTokens as Partial<Record<TextStyles, CSSProperties>>;
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx:16]:
```suggestion
    const designTokens = context.designTokens;
```
The type should already be `DesignTokens`

Comment (SamuelAlev) [inline src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx:16]:
```suggestion
    const { designTokens } = useRichTextEditorContext();;
```
The type should already be `DesignTokens`

File: src/components/RichTextEditor/serializer/utils/reactCssPropsToCss.ts (hunk 1 above)
Comment (mike-schmid) [reply src/components/RichTextEditor/serializer/utils/reactCssPropsToCss.ts]: It's of type `CSSProperties`. I've updated it now. Thanks for the hint 👍

File: src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx (hunk 2 above)
Comment (mike-schmid) [reply src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx]: fixed

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**